scss/styles.scss

```diff
@@ -0,0 +1,7 @@
+.container {
```

Comment:
Where do you use this class ? and why in other file?

Reply:
I have used that class to wrap the content of the each view, it is probable that I will make changes regarding the way to implement it.